src/window.js

```diff
@@ -454,11 +454,22 @@ export default class Window extends EventEmitter {
 
     // DnD functionality
     if (this.attributes.droppable) {
+      /*
+        `this.attributes.droppable` should probably always be an object
+        and never be set to `true`.
+        The following code is only to update the setting for legacy applications.
+      */
+      if (this.attributes.droppable === true) {
+        this.attributes.droppable = {};
+      }
+
+      const {dataTransferProperty = 'files'} = this.attributes.droppable;
       const d = droppable(this.$element, {
```

Comment:
A final nitpick 😅 

This section mutates the `attributes` property, which should be avoided.

This can be rewritten as:

```js
if (this.attributes.droppable) {
  const {dataTransferProperty = 'files'} = this.attributes.droppable === true
    ? {}
    : this.attributes.droppable;

  const d = droppable() //...
}
```

Reply:
My idea was to change the property for future use, so that it will consistently be an object. But this works too 👍

Reply:
I see. This could have been done in the `createAttributes` function that sets up the `attributes` property so that no (unexpected) mutations happens further down the chain, but in this context it doesn't really matter :)